content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md

```diff
@@ -26,7 +26,11 @@ when using kubeadm to set up a kubernetes cluster.
 * Some infrastructure to copy files between hosts. For example `ssh` and `scp`
   can satisfy this requirement.
 
-[toolbox]: /docs/setup/independent/install-kubeadm/
+<<<<<<< HEAD:content/en/docs/setup/production-environment/tools/independent/setup-ha-etcd-with-kubeadm.md
```

Comment:
This looks like a merge left in by mistake.

Reply:
Thank you it was a miss. Fixed it now.